packages/map-template/src/hooks/useMapBoundsDeterminer.js

```diff
@@ -152,11 +147,9 @@ const useMapBoundsDeterminer = () => {
 
 export default useMapBoundsDeterminer;
 
-
 /**
  * Make the map go to specified geometry with optional padding, zoom level, pitch and bearing.
  *
- * @param {string} mapType
  * @param {object} geometry
```

Comment:
Missing zoom level from the docs here.
Also, should we write some docs for each of there params?

Reply:
ZoomLevel is on line 157.
I (or someone not human) will add some descriptions.